tools/isobuild/bundler.js

```diff
@@ -184,6 +185,62 @@ exports.ignoreFiles = [
       */
 ];
 
+const METEOR_IGNORE = '.meteorignore'
+
+/*
+    .meteorignore works *like*
+    https://git-scm.com/docs/gitignore
+    what's implemented: 
+      - blank line separator for readability
+      - A line starting with # as a comment, \# or [#] is not comment.
+      - Trailing spaces are ignored.
+      - glob stars ** matches all directory. 
+        for example trailing /** matches eveying inside   */   
+//       a/**/b matches a/b a/x/b, a/x/y/b...  
+/*
+    what's implemented but not in the specification.
+      - ? for match single character
+      - [] to match character group.
+
+    What's not effective: (doesn't work.)
+      - ! negative pattern won't be effective unless the below known issue is resolved.
+
+    known issue:
+      ignoreFiles take regExp list with default logical *or*, 
+      instead of glob pattern list with optional logical *and* for negative pattern.
+        *        # ignore all files
+        !*.html  # but (AND) not ignore *.html, ignore files that matches * AND !*.html
+        !*.css   # (AND) not ignore *.css
+        !*.js    # (AND) not ignore *.js
+*/
+
+if (files.exists(METEOR_IGNORE)) {
+  var dotIgnoreFiles = files.readFile(METEOR_IGNORE).toString('utf8')
+  exports.ignoreFiles = exports.ignoreFiles.concat( // wanted use array spread operator .push(...dotIgnoreFiles.split( maybe from v1.6
+    dotIgnoreFiles
```

Comment:
@levithomason Oh. I see. No problem. 
Actually, using `ignore` and rewrite the filter make sense to me. The current approach to the use of regexp to filter files I have to convert from .ignore's glob expression to regexp. So, I have to write every exception in that case so bugs and missing features can occur. Those are exactly what `ignore` did already. So using `ignore` and rewrite the filter to be 100% compatible with .gitignore's glob pattern is the better to prevent bugs or minor incompatibility. Just need to rewire more code. In that case, I need to be sure that I don't break anything I don't see because I just briefly read the part of Meteor code to implement the feature I wanted to see. 

The bottom line is that I am really looking forward to seeing this feature to be included so I will look into it. What do you think @benjamn? Should the filter be written from regexp to glob expression?